packages/components/app/styles/components/tabs.scss

```diff
@@ -20,12 +20,14 @@ $indicator-height: 3px;
   display: flex;
   margin: 0;
   padding: 0;
-  list-style: none;
   border-bottom: 1px solid var(--token-color-border-primary);
 }
 
 .hds-tabs__tab {
   position: relative; // for positioning indicator
+  margin: 0;
+  padding: 0;
+  list-style: none;
```

Comment:
because the `Tab` can live as standalone element, I had to move the `list-style` reset here, and reset also the `margin/padding` of the `<li>` element

Reply:
Ok so I guess making the Tab a standalone component is just for the example in the documentation but it does result in invalid HTML. (`li` elements should only be used as children of a `ul`, `ol`, or `menu` parent element.)

Reply:
I've pushed a change, to fix this.

Reply:
Looks good. I will go ahead and merge. Thanks for all your help creating this.